CHANGELOG.md

```diff
@@ -7,7 +7,7 @@ We follow the [Semantic Versioning 2.0.0](http://semver.org/) format.
 -
 
 ### Changed
--
+- **capital-framework:** [PATCH] Standardized all the usage docs for v4
```

Comment:
Thanks for explaining it. I've made that update, so I think we should be all set.